.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
 python:
   - "2.7"
-  - "3.4"
```

Comment:
Instead of removing Python 3 support, can we conditionally install the CloudML and Tensorflow libraries only if the version of Python is Python 2?

I.E. have everything work on Python 2, but if you want Python 3, you can still use the pydatalab library with storage or bigquery.